display/playhead.py

```diff
@@ -0,0 +1,69 @@
+import sys
+import pygame
+sys.path.append('/Users/borisjoens/Dropbox/Kommentare/SmallData')
+# import statements must change if imported from song/song_server.py
+from display.animation import Animation
+from display.status_display import SCREEN_HEIGHT, SONG_WIDTH
+
+
+class Playhead:
+    def __init__(self):
+        self.playhead_surface = pygame.Surface((SONG_WIDTH / 8, SCREEN_HEIGHT / 4))
+        self.playhead_surface.set_colorkey((0, 0, 0))
+        self.playhead = self.create_playhead(self.playhead_surface.get_size())
+        self.anim_part_begin = Animation(self.playhead_surface, 0, 0, 1, self.playhead_surface.get_width(), \
+                                                              self.playhead_surface.get_height(), 1)
+        self.anim_right = Animation(self.playhead_surface, 0, 0, 1, self.playhead_surface.get_width(), \
+                                                              self.playhead_surface.get_height(),1)
+        self.song_parts = {'intro' : [2, 499]}
```

Comment:
das sieht mir danach aus, dass es eigentlich von woanders herkommen muss. Nicht aus dem song-file? Vielleicht sollte man hier noch nen TODO in den code schreiben...